### PR TITLE
 #33: Updated integration-test-enviornment to v0.3 

### LINKED ISSUE #33: Update integration-test-environment to version 0.3
## Background
- integration-test-environment version 0.3 contains new docker-db versions and new commandline parameters to set db-mem size and db-disk size

## Acceptance Criteria
- updated version project.toml and poetry.lock
- add commandline arguments for db-mem size and db-disk size to run-db-test

### PR DESCRIPTION
**Pull request by tomuben**
Fixes #33

### CODE REVIEW

File: test/test_run_db_test_docker_db_check_arguments.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        if environment_info_json_path.exists():
+            with environment_info_json_path.open() as f:
+                return EnvironmentInfo.from_json(f.read())
+
+    def assert_mem_disk_size(self, mem_size: str, disk_size: str):
+        env_info = self._getEnvironmentInfo()
+        containers = \
+            [c.name for c in
+             self.client.containers.list()
+             if env_info.database_info.container_info.container_name in c.name]
```
Comment (tkilias) [inline test/test_run_db_test_docker_db_check_arguments.py:39]: was c.name a list or string, if it is a string, we should compare by equality

File: test/test_run_db_test_docker_db_check_arguments.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        db_container = [c for c in containers if "db_container" in c]
+        exit_result = self.client.containers.get(db_container[0]).exec_run("cat /exa/etc/EXAConf")
```
Comment (tkilias) [inline test/test_run_db_test_docker_db_check_arguments.py:41]: Shouldn't be db_container already a container object?

File: test/test_run_db_test_docker_db_check_arguments.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        if output == '':
+            exit_result = self.client.containers.get(db_container[0]).exec_run("cat /exa/etc/EXAConf")
+            output = exit_result[1].decode("UTF-8")
+            return_code = exit_result[0]
+        return_code = exit_result[0]
+        self.assertEqual(return_code, 0)
+        self.assertIn(f"MemSize = {mem_size}", output)
+        self.assertIn(f" Size = {disk_size}", output)
+
+    def remove_docker_runtime(self):
```
Comment (tkilias) [inline test/test_run_db_test_docker_db_check_arguments.py:52]:
```suggestion
    def remove_docker_environment(self):
```

docker runtime is misleading

**Review by tkilias — CHANGES_REQUESTED** [review]
A few small things and please check why the CI test failed

File: test/test_run_db_test_docker_db_check_arguments.py (hunk 2 above)
Comment (tomuben) [reply test/test_run_db_test_docker_db_check_arguments.py]: Simplified in latest commit.

**Review by tkilias — APPROVED** [review]

**Pull request merged**